cpp/src/cider/tests/substrait_plan_files/between_and_fp32_velox.json

```diff
@@ -89,7 +89,7 @@
           "output_type": {
            "bool": {
             "type_variation_reference": 0,
-            "nullability": "NULLABILITY_REQUIRED"
+            "nullability": "NULLABILITY_NULLABLE"
```

Comment:
Q: what's the difference between `NULLABILITY_NULLABLE` and `NULLABILITY_REQUIRED`

Reply:
```nullable``` and ```not nullable```, [ref](https://substrait.io/tutorial/sql_to_substrait/#types-and-schemas).